src/Guide/Api/Methods.hs

```diff
@@ -58,6 +60,26 @@ createCategory db title' group' = do
       -- TODO addEdit edit
       return catId
 
+-- | Edit categoty's note.
+setCategoryNotes :: DB -> Uid Category -> Text -> Handler NoContent
+setCategoryNotes db catId note = uncache db (CacheCategoryNotes catId) $ do
```

Comment:
The endpoint's description says that it returns 404 when the category is not found, but it does not. Please add checks and `throwError (err404 {errBody = "Category not found"})` to both endpoints (and to all other endpoints where the description says 404 may be returned)